app/models/concerns/individual_market_behaviors.rb

```diff
@@ -0,0 +1,13 @@
+module IndividualMarketBehaviors
+  extend ActiveSupport::Concern
+
+  included do
+
```

Comment:
depending on the situation you may need an instance method to confirm that individual market is enabled